.travis.yml

```diff
@@ -34,3 +35,8 @@ script:
         source ./.CI/build_all;
       fi
 
+cache:
+    directories:
+        - $HOME/cache
+before_cache:
+    - find $HOME/cache/pkgs/ -mindepth 1 -maxdepth 1 -type d -exec rm -r {} \;
```

Comment:
Also @gqmelo this is kind of important. It makes sure we only cache the tarballs of the packages not their decompressed directories.